src/frontend/screens/Game/GamePage/index.scss

```diff
@@ -1,409 +1,731 @@
-.gameConfigContainer {
-  background: var(--background-darker);
-  box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);
-  display: flex;
-  height: 100%;
-  overflow: hidden;
-  justify-content: stretch;
-  position: relative;
-
-  .titleWrapper {
-    display: flex;
-    position: sticky;
-    top: 0px;
-    padding-top: 2rem;
-    margin-top: 2rem;
+.App.oldDesign {
```

Comment:
this diff looks super broken here, but I basically moved all the original style inside `.App.oldDesign` and then added new style after this for the new design, but github is mixing everything